app/helpers/thredded/application_helper.rb

```diff
@@ -17,6 +17,16 @@ def user_link(user)
       render partial: 'thredded/users/link', locals: { user: user }
     end
 
+    # @param user [Thredded.user_class]
+    # @return [String] wrapped @mention string
+    def user_mention(user)
+      if user.to_s.include?(' ')
```

Comment:
Instead of `user.to_s`, this should actually be `user.send(Thredded.user_name_column)`. There is no distinction in the demo, but on a site like GitHub `to_s` would return the full name, but mentions would happen with @-username.